src/lib_ccaligner/commons.cpp

```diff
@@ -4,20 +4,20 @@
  * Link     : https://github.com/saurabhshri
 */
 
-#include "commons.h"
+#include <commons.h>
 
-void ms_to_srt_time(long int ms, int *hours, int *minutes, int *seconds, int *milliseconds)
+void ms_to_srt_time(long int ms, int hours, int minutes, int seconds, int milliseconds)
 {
-    *milliseconds = (int)(ms % 1000);
-    ms = (ms - *milliseconds) / 1000;
+    milliseconds = (int)(ms % 1000);
```

Comment:
It would change the behavior of the program.
Pointers here are intended as we need to change the values of the parameters.
However, you can change them to references.

Reply:
thank you, I completely forgot about pointers, need to review my knowledge on c++ again clearly